trillian/integration/demo-script.sh

```diff
@@ -92,7 +101,7 @@ hammer_pid=$!
 sleep 30
 
 echo 'Remember to kill off all of the jobs, so their (hard-coded) ports get freed up.  Shortcut:'
-./trillian/integration/ct_killall.sh
+${GOPATH}/trillian/integration/ct_killall.sh
```

Comment:
Also doesn't work if GOPATH has multiple entries. Do we care about this case, which is probably not usual?